docs/releasing.md

```diff
@@ -14,9 +14,8 @@ to run something like this:
 
 ```shell
 root=$(jj log --no-graph -r 'heads(tags())' -T 'commit_id')
```

Comment:
comment (off-topic): btw this command is probably wrong now that we have tags without common descendants 🙃 (i.e. `root` might resolve to more than one revision?).

Reply:
I guess we could use `latest(heads(tags))` which should usually return the desired tag.

Reply:
I will defer that for another PR if we decide that's good enoough.